lib/compress.js

```diff
@@ -1838,6 +1838,11 @@ merge(Compressor.prototype, {
     });
 
     OPT(AST_If, function(self, compressor){
+        if (self.alternative instanceof AST_EmptyStatement) {
+            self.alternative = null;
+            // fall through
+        }
+
```

Comment:
The only optimisation before that `is_empty()` line is the `if (const) ...` rule.

But I agree the current fix is of lower risk 👍